ftw_tools/models/delineate_anything.py

```diff
@@ -85,7 +86,7 @@ def polygonize(
         def pixel_to_geo(x, y, z=None):
             return transform * (x, y)
 
-        df = result.to_df()
+        df = result.to_df().to_pandas()
```

Comment:
Do we need to_pandas() here? Isn't that what to_df() is already doing?

Reply:
No, to_df returns a Polars dataframe, which doesn't have the Pandas apply functions used below.

Reply:
They have changed that recently: 
https://community.ultralytics.com/t/new-release-ultralytics-v8-3-187